src/FeatureFlags/OptionBasedFormEditor/Settings/General.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Give\FeatureFlags\OptionBasedFormEditor\Settings;
+
+/**
+ * @unreleased
+ */
+class General extends AbstractOptionBasedFormEditorSettings
+{
+    /**
+     * @unreleased
+     */
+    public function getDisabledOptionIds(): array
+    {
+        return [
+            //General Section
+            'success_page',
+            'failure_page',
```

Comment:
We can keep these (recently added support for success page)

Reply:
Resolved in 512ad4dc6a0e8c13cfb06b4553ddf969ab81e0fd